src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
 // write your code here
+const span = document.querySelectorAll('.population');
+
+const newFormat = new Intl.NumberFormat('en-Us');
+
+const ourList = [...span].map(item => +(item.innerText.split(',').join('')));
+
+const totalPopulation = ourList.reduce((item, sum) => sum + item, 0);
```

Comment:
```suggestion
const totalPopulation = ourList.reduce((sum, item) => sum + item, 0);
```
On the contrary, first value is sum (accumulator). Also, I think item is not the best name for population number.